red/runtime/nodes/Node.js

```diff
@@ -258,6 +316,28 @@ Node.prototype.error = function(logMessage,msg) {
     }
 };
 
+Node.prototype.setCanSendSuccess = function(val) {
+    this._can_send_success = val;
+}
+
+Node.prototype.canSendSuccess = function() {
+    return (this._can_send_success === true);
+}
+
+Node.prototype.maySendSuccess = function() {
+    return (this._can_send_success !== false);
+}
+
+Node.prototype.success = function(msg) {
```

Comment:
Should this be `Node.prototype.done`? (Part of the wider naming of 'success' in this feature - see top level comments)

Reply:
I think `Done` is a good choice for node name.
My concern here is `done` is used as callback name in many places in JavaScript code and may be confusing.